services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/HankeBuilder.kt

```diff
@@ -59,18 +64,8 @@ data class HankeBuilder(
      */
     fun save(): Hanke {
         val createdHanke = create()
-        return hankeService.updateHanke(
-            hanke.copy(id = createdHanke.id, hankeTunnus = createdHanke.hankeTunnus).apply {
-                omistajat = hanke.omistajat.onEach { it.id = null }
-                rakennuttajat = hanke.rakennuttajat.onEach { it.id = null }
-                toteuttajat = hanke.toteuttajat.onEach { it.id = null }
-                muut = hanke.muut.onEach { it.id = null }
-                tyomaaKatuosoite = hanke.tyomaaKatuosoite
-                tyomaaTyyppi = hanke.tyomaaTyyppi
-                alueet = hanke.alueet
-                tormaystarkasteluTulos = hanke.tormaystarkasteluTulos
-            }
-        )
+        hanke.copy(id = createdHanke.id, hankeTunnus = createdHanke.hankeTunnus).apply {}
```

Comment:
Empty `apply`?

Reply:
This whole line didn't do anything. Removed.